src/Cli/InstallCommand.php

```diff
@@ -10,6 +10,14 @@
 
 class InstallCommand extends Command
 {
+    private $dockerInstaller;
+
+    public function __construct(DockerInstaller $dockerInstaller)
+    {
+        $this->dockerInstaller = $dockerInstaller;
+        parent::__construct();
```

Comment:
I'm sorry, but you're wrong.

dock-cli git:(mock-process-runner) bin/dock-cli docker:install
PHP Fatal error:  Uncaught exception 'LogicException' with message 'Command class "Dock\Cli\InstallCommand" is not correctly initialized. You probably forgot to call the parent constructor.' in /Users/tpiper/Documents/webdev/dock-cli/vendor/symfony/console/Application.php:395